nixos/modules/services/misc/redmine.nix

```diff
@@ -266,7 +266,7 @@ in
       environment.REDMINE_LANG = "en";
       environment.SCHEMA = "${cfg.stateDir}/cache/schema.db";
       path = with pkgs; [
-        imagemagick
+        imagemagickBig
```

Comment:
Does redmine uses features that are only available in `imagemagickBig`?

Reply:
Sorry. My earlier research convinced me that this was the case, but further googling revealed that all Redmine plugins that use ghostscript expect `gs` to be in PATH. For these cases this fix has no effects. 
I'm closing this now.